python/test/conftest.py

```diff
@@ -26,16 +29,141 @@ def eyes_runner_class():
+                capabilities = {
+                    'browserName': 'internet explorer',
+                    'browserVersion': '11.285',
+                    'platformName': 'Windows 10',
+                    }
+                driver = webdriver.Remote(command_executor=sauce_url, desired_capabilities=capabilities)
+                break
+            if browser_type == "Edge":
+                capabilities = {
+                    'browserName': 'MicrosoftEdge',
+                    'browserVersion': '18.17763',
+                    'platformName': 'Windows 10',
+                    'screenResolution': '1920x1080',
+                    }
+                driver = webdriver.Remote(command_executor=sauce_url, desired_capabilities=capabilities)
+                break
+            if browser_type == "Safari11":
+                if (legacy):
+                  capabilities = {}
+                  capabilities['browserName'] = "safari"
+                  capabilities['platform'] = "macOS 10.12"
+                  capabilities['version'] = "11.0"
+                else:
+                  capabilities = {
+                    'browserName': 'safari',
+                    'browserVersion': '11.0',
+                    'platformName': 'macOS 10.12',
+                  }
+                driver = webdriver.Remote(command_executor=sauce_url, desired_capabilities=capabilities)
+                break
+            if browser_type == "Safari12":
+                if (legacy):
+                  capabilities = {}
+                  capabilities['browserName'] = "safari"
+                  capabilities['platform'] = "macOS 10.13"
+                  capabilities['version'] = "12.1"
+                else:
+                  capabilities = {
+                    'browserName': 'safari',
+                    'browserVersion': '12.1',
+                    'platformName': 'macOS 10.13',
+                  }
+                driver = webdriver.Remote(command_executor=sauce_url, desired_capabilities=capabilities)
+                break
+            if browser_type == "ChromeEmulator":
+                mobile_emulation = {
+                        "deviceMetrics": { "width": 384, "height": 512, "pixelRatio": 2.0 },
+                        "userAgent": "Mozilla/5.0 (Linux; Android 8.0.0; Android SDK built for x86_64 Build/OSR1.180418.004) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/69.0.3497.100 Mobile Safari/537.36" 
+                      }
+                options.add_experimental_option("mobileEmulation", mobile_emulation)
+                options.add_argument("--headless")
+                driver = webdriver.Chrome(executable_path=ChromeDriverManager().install(), options=options,)
+                break
+            if browser_type != "Chrome" and browser_type != "Firefox" and browser_type != "IE11" and browser_type != "Edge" and browser_type != "Safari11" and browser_type != "Safari12" and browser_type != "Appium": raise ValueError           
```

Comment:
More elegant way here would be:

`if browser_type not in ["Chrome", "Firefox", "IE11", "Edge", "Safari11", "Safari12", "Appium"]:`